services/minespace-web/package.json

```diff
@@ -182,5 +185,37 @@
   },
   "browserslist": [
     "> 0.1%"
-  ]
+  ],
+  "husky": {
```

Comment:
shouldn't be necessary, new husky version runs hooks from .husky folder in repo root